panel/interact.py

```diff
@@ -447,6 +447,14 @@ def square(num=2):
             def square(num=2):
                 print("{} squared is {}".format(num, num*num))
         """
+        for k, v in kwargs.items():
+            if isinstance(v, param.parameterized.Parameter):
+                pass
+            elif isinstance(v, Widget):
+                kwargs[k] = v.param.value
```

Comment:
I actually thought interact already accepted widgets. Is that not the case?

Reply:
On second look I believe it is the case. How silly of me.